README.md

```diff
@@ -59,4 +59,15 @@ You can replace `[tag]` by either a commit hash (the first 10 characters) or by
 
 ## Jar downloads
 
-Jar downloads are available from the [release section](https://github.com/Together-Java/TJ-Bot/releases).
\ No newline at end of file
+Jar downloads are available from the [release section](https://github.com/Together-Java/TJ-Bot/releases).
```

Comment:
This is good stuff, but it should go into our wiki page regarding **Contributing**. Please move it over to there :+1: